WANTED.md

```diff
@@ -16,6 +16,7 @@ to add this, please file an issue to discuss an implementation path.)
 * Lisp's [CL-PPCRE](https://github.com/edicl/cl-ppcre/).
 * A selected subset of the [mess that is regex libraries for
 Haskell](https://wiki.haskell.org/Regular_expressions).
+* An hybrid pyahocorasick (in C) and Python multiple regex engine https://github.com/Quantco/multiregex by @jonashaag
```

Comment:
It looks like this is "just" a wrapper over `pyahocorasick` and Python's standard library `re` module: https://github.com/Quantco/multiregex/blob/d1209b6f68f2037d5a1b10c6ed8b96af78619c4b/src/multiregex/__init__.py

It probably isn't worth adding to this barometer, but I could be convinced otherwise if it achieved broad use.

I don't want to reject adding this at all since this list is already pretty aspirational. But I don't think it belongs here. Could you add it to the list below and note my concerns with it?

Also, please remove the author's name. Nothing against it, but I didn't write out the authors for any of the other regex engines, so I'm not sure why we should do it for this one. Author information is often complex anyway, and I'd rather not sign up to maintain it.